web/src/app/dialogs/new-inspection/components/service/parameter-store.ts

```diff
@@ -100,12 +102,31 @@ export class DefaultParameterStore implements ParameterStore {
       });
   }
 
+  /**
+   * Update the default values for parameters.
+   * If the current value is same as the previous default values, the parameter is updated with the newer default values.
+   * If not, the parameter value is kept because it was updated by the user.
+   */
   setDefaultValues(defaultValues: { [id: string]: unknown }): void {
     this.currentParameters
       .pipe(takeUntil(this.destroyed), take(1))
       .subscribe((parameters) => {
-        this.currentParameters.next({ ...defaultValues, ...parameters });
+        const wasDefault: { [id: string]: boolean } = {};
```

Comment:
From the perspective of design, letting the data object which represents the state of the form have a property like `dirty` in `AbstractControl` (https://angular.dev/api/forms/AbstractControl#dirty) and use them as a condition might be cleaner. 
At the same time, that require more change and I am certainly not expert on this frontend architecture so the above approach could be imfeasible in the first place. So you can disregard it if it's not applicable. Just dropping the thought.

Reply:
I added the dirty observable. And it's also needed to show the status of the field on the form field headers.